.github/actions/config-git/action.yml

```diff
@@ -0,0 +1,18 @@
+name: "config-git"
+description: "Configure git"
+inputs:
+  branch-name:
+    description: "Branch name to checkout"
+    required: true
+  user-email:
+    description: "Email to link commits to user"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: git checkout -B ${{ inputs.branch-name }}
+    - shell: bash
+      run: git fetch
```

Comment:
checkout is usually done best via the standard actions/checkout

in this action I would just address the user.email and user.name configuration (so I would call it something like configure-git-author)

Reply:
Done.